tagger.go

```diff
@@ -68,22 +78,36 @@ func checkLinodeTagsAgainstConfig(linodes []linodego.Instance, rules []TagRule)
 
 				// check `absent` tags to remove unwanted tags
 				for _, tag := range linode.Tags {
+					report[tag] = map[string]string{}
 					if !slices.Contains(rule.Tags.Absent, tag) {
 						// if this tag is not on the `absent` list,
 						// we can persist it through to the new tag set
 						newTags = append(newTags, tag)
+					} else {
+						// if this tag *is* on the `absent` list, we'll be removing it later
+						absentReportCounter += 1
+						report[tag]["absent"] = strconv.Itoa(absentReportCounter)
+						// use a list to capture updated LinodeIDs that later get shoved into a map value
+						linodeList = append(linodeList, linode.Label)
```

Comment:
If we're flattening this list down into a comma separated list after anyway, this could be made a bit more efficient using a string builder:
https://pkg.go.dev/strings#Builder
https://www.calhoun.io/concatenating-and-building-strings-in-go/

Reply:
39e2a3ccc74ca91bd01fe06f02a3c6d23464f2df